[
    {
        "content": "<p>wrbs <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739#issuecomment-802139811\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739\">Issue #2739</a>:</p>\n<blockquote>\n<p>It looks like these test failures are unrelated to the change</p>\n</blockquote>",
        "id": 230904236,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1616087982
    },
    {
        "content": "<p>cfallin <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739#issuecomment-802142056\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739\">Issue #2739</a>:</p>\n<blockquote>\n<p>It looks like the CI failure happened on the merge commit of #2723 yesterday (though the CI on the PR itself was green) -- possibly related? cc @bnjbvr / @alexcrichton </p>\n</blockquote>",
        "id": 230904786,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1616088159
    },
    {
        "content": "<p>alexcrichton <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739#issuecomment-802152864\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2739\">Issue #2739</a>:</p>\n<blockquote>\n<p>There's a bit more discussion about CI on <a href=\"https://github.com/bytecodealliance/wasmtime/pull/2736#issuecomment-801552884\">https://github.com/bytecodealliance/wasmtime/pull/2736#issuecomment-801552884</a>, I haven't had a chance to dig in yet myself</p>\n</blockquote>",
        "id": 230907537,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1616089098
    }
]